credential prompt (#2710)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md

## Step 3: Configure Exchange Online users to access Exchange Server on-premises public folders

> A Microsoft 365 or Office 365 mailbox who is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access on-premises public folders.  

Reviewer: ```suggestion
A Microsoft 365 or Office 365 mailbox that is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access on-premises public folders.  
```

You can use following command to identify such mailboxes:

```PowerShell
Get-Mailbox |?{$_.IsDirSynced -eq $false}
```

> The above users will keep getting credential prompt after public folder mailboox access is configured. Use one of the following solution for such users, before enabling public folder access:

Reviewer: ```suggestion
These users will keep getting credential prompts after public folder mailbox access is configured. Use one of the following solutions for such users before enabling public folder access:
```

> 1) Link the EXO only mailboxes, listed in step above, to On-Premises users by using the Knowledge Base article [Exchange Online users can't access legacy on-premises public folders](https://support.microsoft.com/help/3106618).

Reviewer: ```suggestion
1. Link the Exchange Online only mailboxes listed in the previous step to on-premises users by using the steps described in [Exchange Online users can't access legacy on-premises public folders](https://support.microsoft.com/help/3106618).
```

> 2) Use steps provided in [Controlled Connections to Public Folders](https://aka.ms/ControlPF) to enable public folder access only to mailboxes that have linked users at On-Premises.

Reviewer: ```suggestion
2. Use the steps provided in [Controlled Connections to Public Folders](https://aka.ms/ControlPF) to enable public folder access only to mailboxes that have linked users on-premises.
```

> 

Reviewer: ```suggestion
```
